mantidimaging/gui/windows/main/view.py

```diff
@@ -563,8 +561,9 @@ def closeEvent(self, event) -> None:
             # Close additional windows which do not have the MainWindow as parent
             if self.recon:
                 self.recon.close()
-            if self.live_viewer:
-                self.live_viewer.close()
+            if self.live_viewer_list:
+                while self.live_viewer_list:
+                    self.live_viewer_list[-1].close()
```

Comment:
You could remove the `if self.live_viewer_list:`, the while will do nothing if the list is empty.